examples/flutter_login/lib/login/bloc/login_state.dart

```diff
@@ -25,4 +25,9 @@ class LoginState extends Equatable {
 
   @override
   List<Object> get props => [status, username, password];
```

Comment:
After decoupling purity, validity, and submission status. This will be the appropriate way to define equatable's props
```dart
List<Object> get props => [status, username, password, isValid, isPure];
```